### PR TITLE
The Next Web page clone

### PR DESCRIPTION
**Pull request by KinezaEnaelle**
This project is about cloning The Next Web Page using HTML and CSS. Float, flex, and grid are highly used. In this project, we use media queries to respond to the responsiveness of the page through 2 breakthroughs.

### CODE REVIEW

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet. You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [x] Please use the same linter as present [here](https://github.com/microverseinc/linters-config/tree/master/html-css). 

- [x] Please add these two folders in one folder. This can help you present your project more impressively. 
chrome-capture (22)

- [x] Instead of using the same code for each item, please make it peculiar. 
chrome-capture (24)

- [x] on the medium screen, the logo is not aligned correctly. 
chrome-capture (25)

- [x] on the medium screen, the bottom text should be opposite to the top one. 
chrome-capture (26)
- [x] on the small screen, the logo is not aligned correctly. 
chrome-capture (25)

- [x] on the small screen, the bottom text should be opposite to the top one. 
chrome-capture (26)



- [x] For each content please use a different image. This will help you to attract the audience. 
**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Comment by arslanbisharat** [comment]
Hello Team I made a mistake in your project. I didn't mean you to remove the deployment part from your readme. I meant was testing. Please work on it.
I am sorry for the trouble.
Regards,
Arslan Bisharat

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet. You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [x] Please work on the logo in medium and small screens. The logo is not aligned correctly. 
chrome-capture (29)

- [x] On small screen and tablet mode, the footer logo is not aligned correctly. 
chrome-capture (32)


- [x] Text is over-flowing on small screen and tablet mode. 
chrome-capture (31)


- [x] On the small screen, content is not aligned correctly for the whole page. 
chrome-capture (30)



- [x] On the scrolling, the sticky navbar is not as present on the original page.

- [x] As described earlier, please change titles names as present on the original page. 
chrome-capture (28)



**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I must say, you've done an amazing job so far. However, there are some changes that are required to be fixed before this project can be approved. Please check my points below for the required changes and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

_This review is based on the fact that the main point of this project is to implement a Responsive Web Design._

- [x] You're having a bug in your `Latest Deals` section.

Original page:


Your Clone:


**On medium screen sizes (768px - 1022px)**
- [x] The TNW logo is still not placed correctly. Please check the image below, observe the difference, and fix as appropriate.

Original page:


Your clone:


**On Small screen sizes (<768px)**
- [x] There seems to be some layout issue in the `Latest Funding` section.

| Original | Clone |
| --- | ----------- |
|  |  |

- [x] something is definitely wrong with your footer section. 

| Original | Clone |
| ---- | ---------- |
|  |  |

- [x] I can see that you've added the TNW logo at the footer section. You still have some alignment issues. Kindly check and fix it.

| Original page | Your Clone page |
| ------ | ----------- |
|  |  |


**[ COMMENTS ]**
- I observed that you are working on the project while reviewing. To have the best review experience from TSE's, please do not push changes to a project you've already submitted. You can make and commit the changes locally, then wait till you receive the feedback before pushing the changes.

**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I'm so happy to see that you have carefully implemented the previous changes identified. However, we still have some little fixes to make.__

_It's okay, this is a large project so it's possible that one change will lead to another. Do not give up, you are almost there :muscle:_

**[ REQUIRED CHANGES ]**

_We need to be sure that nothing is messed up on different screen sizes_

- [x] From the previous change you made in the `Latest Funding` Section, the `|` symbol should not be rendered for on small devices.

| Original | Your Clone |
| ----- | ------------ |
|  |  |

- [x] The TNW logo is missing a part on small screen sizes. 

| Original | Your Clone |
| ----- | ------------ |
|  |  |

- [x] Add more spaces within each section on the small screen sizes. I will point out one instance, please apply it to other sections of the page. Adding the white spaces makes your page more organized.

| Original | Your Clone |
| ----- | ------------ |
|  |  |

- [x] You are missing a part in each of the articles on the `Latest Deals` section.

| Original | Your Clone |
| ----- | ------------ |
|  |  |

**[ COMMENTS ]**
- Compare your page with the original page at all screen sizes, observe the differences and try to customize your page so it looks similar to the original.
- Your page seems to be messed up when rendered on very small screen sizes (<=400px). Do try your best to make it look good in all screen sizes.
- Pay attention to the little details like font size, margins, padding, and colors. They are an essential part of the page.
- Do not be discouraged. These reviews are to ensure that you build a professional website that you can be proud to show anyone in the future.

**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

File: index.html
Comment (ghost) [inline index.html]: **Wrap these sections inside the `header` tag.**

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

### Hello team,

**Great job implementing the changes requested. However, there a few fixes needed.**

1). **How you added more spaces within each section on the small screen sizes, please do the same for the medium and large screens.**

2). **The TNW logo should display on the navbar as I scroll the page for the desktop screen sizes(> 1024px).**

original | clone
---------| -------
tnwnav_original | tnwnav_clone

3). **On 768px width, there is some cut off text on the rightmost side of the black navbar.**

tnwtopnav_clone
4). **Check the comment below.**

## Optional

**Please note latest funding section displays three cards from 1024px and not 1023px. Also, in the same section, the subtitles display on multiple lines from 768px and not 767px.**

### Happy coding 😃.

**You can reach me on slack (Rose Wanjohi).**

File: index.html
Comment (ghost) [reply index.html]: The sticky element will stick only to the parent element, so here it should be a child of body, not header

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

#### Hello @KinezaEnaelle and @Igors78, 
You've done an awesome job so far 👌. Keep up the good work. 
Let us quickly add some finishing touches and request another review.

- [x] At a width of about 900px below, there're huge gaps between each section, kindly close up these spaces. The screenshot below shows some of the gaps at 600px screen width.
gap

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by ghost — APPROVED** [review]
# Status: Approved

**Congratulations team and good luck on your next project 😃.**

**Pull request merged**
